nautobot/circuits/models.py

```diff
@@ -240,7 +240,10 @@ class Meta:
         unique_together = ["circuit", "term_side"]
 
     def __str__(self):
-        return "Side {}".format(self.get_term_side_display())
+        return "{} - Side {}".format(self.circuit.cid, self.term_side)
+
+    def get_absolute_url(self):
+        return reverse("circuits:circuit", args=[self.circuit.pk])
```

Comment:
Let's try to make this forwards-compatible with the changes coming in 1.3.0 as a result of #1144 - I think the below might be a step in the right direction?
```suggestion
        return f"Termination {self.term_side}: {self.site}"

    def get_absolute_url(self):
        return self.site.get_absolute_url()
```